plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFileFormats.java

```diff
@@ -376,12 +376,16 @@ public void testIon(int rowCount, long fileSizePadding)
                 .filter(tc -> !(tc.type instanceof MapType))
                 .collect(toList());
 
+        HiveConfig hiveConfig = new HiveConfig();
```

Comment:
Can we just use the above `TestHiveConfig`?

Reply:
Did you mean to use the expected version of `HiveConfig` from `TestHiveConfig`. I just used a local `HiveConfig` to keep it consistent with other tests in this flle.

Reply:
I'm sorry I misunderstood what that does. If this is consistent with others than it's good-to-go.